ocs_ci/ocs/dr/dr_workload.py

```
@@ -1146,6 +1146,27 @@ def __init__(self, **kwargs):
         self.preferred_primary_cluster = config.ENV_DATA.get(
             "preferred_primary_cluster"
         ) or (get_primary_cluster_config().ENV_DATA["cluster_name"])
+        dr_cluster_relations = config.MULTICLUSTER.get("dr_cluster_relations", [])
+        if dr_cluster_relations:
+            # dr_cluster_relations is a list containing cluster pairs list
+            dr_cluster_names = dr_cluster_relations[0]
+            self.preferred_secondary_cluster = [
+                cluster_name
+                for cluster_name in dr_cluster_names
+                if cluster_name != self.preferred_primary_cluster
+            ][0]
```

Ruff (0.16.1)

[warning] Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)